Porting/perldocstyle.pod

```diff
@@ -0,0 +1,1117 @@
+own core software repository, and that already covers them under the
+same copyright and license terms as Perl itself. You do not need to
+include additional "LICENSE" or "COPYRIGHT" sections of your own.
+
+These man pages may optionally credit their primary author, or include a
+list of significant contributors, under "AUTHOR" or "CONTRIBUTORS"
+headings. Note that the presence of authors' names does not preclude a
+given page from L<writing in a voice consistent with the rest of Perl's
+documentation|/The documentation speaks with one voice>.
+
+Note that these guidelines do not apply to the core software modules
+that ship with Perl. These have their own standards for authorship and
+copyright statements, as found in C<perlpodstyle>.
+
+=head2 Formatting rules
+
+=head3 Line length and line wrap
+
+Each line within a Perl man page's Pod source file should measure 72
+characters or fewer in length.
+
+Please break paragraphs up into blocks of short lines, rather than
+"soft wrapping" paragraphs across hundreds of characters with no line
+breaks.
+
+=head3 Code blocks
+
+Just like the text around them, all code examples should be as short and
+readable as possible, displaying no more complexity than absolutely
+necessary to illustrate the concept at hand.
+
+For the sake of consistency within and across Perl's man pages, all
+examples must adhere to the code-layout principles set out by
+L<C<perlstyle>|perlstyle>.
+
+Sample code should deviate from these standards only when necessary:
+during a demonstration of how Perl disregards whitespace, for example,
+or to temporarily switch to two-column indentation for an unavoidably
+verbose illustration.
+
+You may include comments within example code to further clarify or label
+the code's behavior in-line. You may also use comments as placeholder
+for code normally present but not relevant to the current topic, like
+so:
+
+    while (my $line = <$fh>) {
+        #
+        # (Do something interesting with $line here.)
+        #
+    }
+
+Even the simplest code blocks often require the use of example
+variables and subroutines, L<whose names you should choose with
+care|/Use meaningful variable and symbol names in examples>.
+
+=head3 Inline code and literals
+
+Within a paragraph of text, use C<CE<lt>...E<gt>> when quoting or
+referring to any bit of Perl code--even if it is only one character
+long.
+
+For instance, when referring within an explanatory paragraph to Perl's
+operator for adding two numbers together, you'd write "C<CE<lt>+E<gt>>".
+
+=head3 Function names
+
+Use C<CE<lt>...E<gt>> to render all Perl function names in monospace,
+whenever they appear in text.
+
+Unless you need to specifically quote a function call with a list of
+arguments, do not follow a function's name in text with a pair of empty
+parentheses. That is, when referring in general to Perl's C<print>
+function, write it as "C<print>", not "C<print()>".
```

Comment:
The text styling should help to disambiguate English verb from function name.

As for parens/non-parens, this is me following apparent precedent set down by existing core documentation, especially `perlfunc`, though the whole collection of Perl documentation is very much of two minds about this.

I do think there should be a standard here, one way or the other. I do prefer the less verbose way, using monospace styling instead of `()` to signal "this is a function". It's less verbose--and it avoids the uncomfortable implication of that the function in question can always be called with no arguments.

Reply:
FWIW I’ve never made that assumption—that a function with empty-parens can implicitly be called without arguments—in Perl any more than I’ve made it in C, where man pages frequently do follow function names with empty parens.

Reply:
Since you're not the first person to question this, let's consider it another research point for the near future. (Albeit a fairly simple one, I expect... a matter of seeing how other project style guides tend to handle the question, and doing whatever that is, adapted for Perl.)

Reply:
Monospace styling doesn't help when it's all monospaced, in a terminal, reading a manpage.